docs/content/api/module/retry.md

```diff
@@ -20,7 +33,9 @@ const circuit = new Circuit({
       // Creates a bulkhead module
```

Comment:
I know it's not part of your changes, but this should be `// Creates a retry module`

Reply:
Will update this also